language/tools/move-cli/src/test.rs

```diff
@@ -10,6 +10,7 @@ use std::{
     path::Path,
     process::Command,
 };
+use walkdir::WalkDir;
```

Comment:
It would be nice to avoid taking this dependency on `walkdir` if we can. I think we use `datatest::iterate_directory` for something similar to this [elsewhere](https://github.com/libra/libra/blob/master/language/stdlib/src/main.rs#L79); maybe that will work here?

Reply:
I think we can avoid this`walkdir`. Let me emulate its functionality.